### DISCUSSION TITLE
auto load collection

### DISCUSSION THREAD

**Discussion by wikoci** [payloadcms/payload#774] [Feature Requests & Ideas]
Request to load all collections automatically,
it's so hard to import each collection evry time in payload.config.

**Comment by DanRibbens** [comment]
Hey wikoci, that is a cool idea but there are some hurdles to think about.

This probably doesn't belong in the core of Payload. Everyone is going to scaffold their projects slightly differently. There are some hurdles to implementing this due to how webpack needs to access files. The configuration is shared both on the browser side and the server side so it isn't as easy as just pulling in collections dynamically.

**Comment by wikoci** [comment]
Hey .  of course.

I did it , last night. :)
payload is flexible and easy to customize .
